i18n/en.yaml

```diff
@@ -9,6 +9,9 @@
 - id: meta_categories
   translation: "Categories"
 
+- id: meta_tags
+  translation: "Tags"
```

Comment:
TODO: Add translations for other supported languages (fr, it, pt, pt-br).

I think you can take the translation from my other theme. See https://github.com/Vimux/Mainroad/tree/master/i18n

Reply:
Added.